src/asymmetric_crypto_ctx.cpp

```diff
@@ -472,11 +517,11 @@ namespace {
         auto r = _BN_bin2bn(signature.data(), keySizeBytes);
```

Comment:
Can you please have a quick look, if moving this function to the impl class would resolve https://asc.bmwgroup.net/mgujira/browse/MGUSEC-4419?
If so I would favour to do so here. :)
But please clarify with Clemens first, if he wants the ticket merging :D

Reply:
tbh, the ticket also says:
'Moreover, pimpl classes (or rather data-structs) use multiple inheritance in the presense of data members, which I would generally consider to be discouraged.'
This is a bigger change which i would like to see in a different commit. I've just fixed the typos here...